go/libraries/doltcore/doltdb/commit_hooks_test.go

```diff
@@ -104,7 +105,16 @@ func TestPushOnWriteHook(t *testing.T) {
 
 	names, err := root.GetTableNames(context.Background(), DefaultSchemaName)
 	assert.NoError(t, err)
-	if len(names) != 0 {
+
+	// ignore dolt_ci_* tables
+	filtered := make([]string, 0)
```

Comment:
Seems like there shouldn't be any dolt_ci tables here if you've never inserted into any